repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasBusinessMetadataDefStoreV2.java

```diff
@@ -107,10 +98,9 @@ public void validateType(AtlasBaseTypeDef typeDef) throws AtlasBaseException {
         AtlasBusinessMetadataDef businessMetadataDef = (AtlasBusinessMetadataDef) typeDef;
 
         //validate uniqueness of display name for BM
-        AtlasVertex ret = typeDefStore.findTypeVertexByDisplayName(
-                businessMetadataDef.getDisplayName(), DataTypes.TypeCategory.BUSINESS_METADATA);
-        if (ret != null && (
-                businessMetadataDef.getGuid() == null || !businessMetadataDef.getGuid().equals(ret.getProperty(Constants.GUID_PROPERTY_KEY, String.class)))) {
+        AtlasBusinessMetadataType ret = typeRegistry.getBusinessMetadataTypeByDisplayName(businessMetadataDef.getDisplayName());
```

Comment:
Possible to cover this with an unit test??
If we don't have any existing tests, we can add one

Reply:
@anbarasantr Yes. Right now we don't cover this via Unit test. But should be able to do. Will add